### PR TITLE
add

### PR DESCRIPTION
**Pull request by alexusernamebruh**
add custom no options selected place holder, default to 'No options selected'
image

### CODE REVIEW

File: src/components/MultiSearchInput/index.tsx [hunk 1]
```diff
@@ -102,7 +104,13 @@ export const MultiSearchInput: React.FC<MultiSearchInputProps> = ({
             <Combobox.Input
               className='w-full rounded-md border border-gray-300 bg-white py-2 pl-3 pr-10 shadow-sm focus:border-THEME-500 focus:outline-none focus:ring-1 focus:ring-THEME-500 sm:text-sm'
               onChange={(event) => setFilter(event.target.value)}
-              displayValue={(v) => `${v?.length || 'No'} options selected`}
+              displayValue={(v) =>
```
Comment (ssysm) [inline src/components/MultiSearchInput/index.tsx]: strongly infer, `v`. If `v` is null have a proper handling case.

**Review by ssysm — CHANGES_REQUESTED** [review]

File: src/components/MultiSearchInput/index.tsx [hunk 2]
```diff
@@ -22,6 +22,15 @@ export interface MultiSearchInputProps {
   noOptionsPlaceholder?: string;
 }
 
+interface displayValue {
```
Comment (mxdi9i7) [inline src/components/MultiSearchInput/index.tsx]: this interface is not called displayvalue, it's actually the DropdownData, also you wouldn't include `length` in its interface

File: src/components/MultiSearchInput/index.tsx [hunk 3]
```diff
@@ -102,7 +104,13 @@ export const MultiSearchInput: React.FC<MultiSearchInputProps> = ({
             <Combobox.Input
               className='w-full rounded-md border border-gray-300 bg-white py-2 pl-3 pr-10 shadow-sm focus:border-THEME-500 focus:outline-none focus:ring-1 focus:ring-THEME-500 sm:text-sm'
               onChange={(event) => setFilter(event.target.value)}
-              displayValue={(v) => `${v?.length || 'No'} options selected`}
+              displayValue={(v: DropdownData) =>
+                `${
+                  v?.length
```
Comment (mxdi9i7) [inline src/components/MultiSearchInput/index.tsx:109]: dropdown data doesn't have a length property. it's not an array

**Review by mxdi9i7 — CHANGES_REQUESTED** [review]
pls fix

**Pull request closed without merging**